deploy.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+
+# Note that this does not use pipefail
+# because if the grep later doesn't match any deleted files,
+# which is likely the majority case,
+# it does not exit with a 0, and I only care about the final exit.
```

Comment:
```suggestion
# Note that this does not use pipefail
# because if the grep later doesn't match any deleted files,
# which is likely to be the case the majority of the time,
# it does not exit with 0, as we're interested in the final exit.
```

@mukeshpanchal27 this is a _very_ nit pick comment, but we should not be talking in a _first person_ style within comments.

Reply:
Thanks @10upsimon. This is not something we will need to handle since this is upstream code from the 10up deploy action and this file will be removed. That said, I agree with your comment and something that we could address upstream.

Reply:
Thank you, @10upsimon and @joemcgill, for the comment. I have an open PR in upstream: https://github.com/10up/action-wordpress-plugin-deploy/pull/120.